src/credentials/CHIPCert.cpp

```diff
@@ -753,6 +764,29 @@ CHIP_ERROR ChipDN::GetCertChipId(uint64_t & chipId) const
     return CHIP_NO_ERROR;
 }
 
+CHIP_ERROR ChipDN::GetCertFabricId(uint64_t & fabricId) const
+{
+    uint8_t rdnCount = RDNCount();
+
+    fabricId = 0;
+
+    for (uint8_t i = 0; i < rdnCount; i++)
+    {
+        switch (rdn[i].mAttrOID)
+        {
+        case kOID_AttributeType_ChipFabricId:
+            VerifyOrReturnError(fabricId == 0, CHIP_ERROR_WRONG_CERT_TYPE);
```

Comment:
Why this condition? How could it occur?

Reply:
This method will retrieve the Certificate Fabric ID. At the same time, that condition will check if there is only one FabricId present in the Certificate. If there's two or more, then the method will fail.

Reply:
```suggestion
            // Ensure only one FabricID RDN present, since start value is 0, which is reserved and never seen.     
            VerifyOrReturnError(fabricId == 0, CHIP_ERROR_WRONG_CERT_TYPE);
```

Reply:
You resolved the comment without applying the suggestion of adding the comment.

Reply:
Good catch, missed this. Included now.